_pages/index.md

```diff
@@ -17,22 +17,24 @@ permalink: /
     <div class="clearfix mxn2">
       <div class="col sm-col-4 px2 mb3">
         <img alt="" src="{{ site.baseurl }}/assets/img/users.svg" height="90">
-        <h2 class="mt2 mb2 pb2 blue border-bottom border-light-blue">Simple and secure for our users</h2>
-        <p>Through login.gov, users can sign in to multiple government agencies. The goal: make managing benefits, services, and applications easier, faster, and more secure.</p>
-        <p>The system replicates the design, performance, and experience people are used to having when logging into their private email or any top tech industry website.</p> 
+        <h2 class="mt2 mb2 pb2 blue border-bottom border-light-blue">Simple *and* secure for the public</h2>
+        <p>Because login.gov is a shared service,</p>
+        <p>* Users need fewer passwords and learn fewer interfaces
```

Comment:
Is the goal to format this as a bulleted list? (it is not formatted as a list since this is HTML not Markdown)